admissions/templates/deleted_application.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block content %}
+En søknad til opptak {{ admission_title }} har blitt slettet.
```

Comment:
Could this also include the group name? Some admissions-admins might be responsible for multiple groups.

Reply:
Sure! Here it is :)
![image](https://github.com/webkom/admissions/assets/90713290/ffd581a5-f22c-4df9-85c6-b08653a2f62a)
Now it sends separate e-mails to each group with their group names.